platform/ext/target/nordic_nrf/common/nrf54l/nrf54l_init.c

```diff
@@ -7,6 +7,8 @@
 #include <stdint.h>
 #include <nrfx.h>
 #include <hal/nrf_oscillators.h>
+#include <hal/nrf_power.h>
```

Comment:
nrf_power.h  not needed?

Reply:
You are right. Leftover from when I was doing testing. I am adding this downstream and then I will create a proper [from tree] afterwards